spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/TaskExecutionsDocumentation.java

```diff
@@ -147,52 +175,46 @@ public void listTaskExecutionsByName() throws Exception {
 						pathParameters(
 								parameterWithName("id").description("The id of an existing task execution (required)")
-						),
-						responseFields(
-								fieldWithPath("executionId").description("The id of the task execution"),
-								fieldWithPath("exitCode").description("The exit code of the task execution"),
-								fieldWithPath("taskName").description("The task name related to the task execution"),
-								fieldWithPath("startTime").description("The start time of the task execution"),
-								fieldWithPath("endTime").description("The end time of the task execution"),
-								fieldWithPath("exitMessage").description("The exit message of the task execution"),
-								fieldWithPath("arguments").description("The arguments of the task execution"),
-								fieldWithPath("jobExecutionIds").description("The job executions ids of the task execution"),
-								fieldWithPath("errorMessage").description("The error message of the task execution"),
-								fieldWithPath("externalExecutionId").description("The external id of the task execution"),
-								fieldWithPath("taskExecutionStatus").description("The status of the task execution"),
-								fieldWithPath("parentExecutionId").description("The id of parent task execution, " +
-										"null if task execution does not have parent"),
-								subsectionWithPath("_links.self").description("Link to the task execution resource")
-						)
-				));
+						)));
 	}
 
 	@Test
-	public void launchTaskCurrentCount() throws Exception {
+	public void taskExecutionRemove() throws Exception {
+
+		documentation.dontDocument( () -> this.mockMvc.perform(
+				post("/tasks/executions")
+						.param("name", "taskB")
+						.param("properties", "app.my-task.foo=bar,deployer.my-task.something-else=3")
+						.param("arguments", "--server.port=8080 --foo=bar"))
+				.andExpect(status().isCreated()));
+
 		this.mockMvc.perform(
-				get("/tasks/executions/current"))
+				delete("/tasks/executions/{ids}?action=CLEANUP", "1"))
 				.andDo(print())
 				.andExpect(status().isOk())
 				.andDo(this.documentationHandler.document(
-                    responseFields(
-						fieldWithPath("[].name").description("The name of the platform instance (account)"),
-						fieldWithPath("[].type").description("The platform type"),
-                        fieldWithPath("[].maximumTaskExecutions").description("The number of maximum task execution"),
-                        fieldWithPath("[].runningExecutionCount").description("The number of number execution")
-                    )
-                ));
+						requestParameters(parameterWithName("action").description("Optional. Defaults to: CLEANUP.")),
+						pathParameters(parameterWithName("ids")
+								.description("The id of an existing task execution (required). Multiple comma separated values are accepted."))
+				));
 	}
 
 	@Test
-	public void removeTaskExecution() throws Exception {
+	public void taskExecutionRemoveAndTaskDataRemove() throws Exception {
```

Comment:
The "Request Structure", "Example Request", "Response Structure" for task execution delete is not showing up in the generated docs.     You have to go to the api-guide.adoc and update the name (remove-task-execution) of that section to task-execution-remove-and-task-data-remove.